python/benchmarks/parse_bdd100k.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+"""Parse Berkley 100K dataset."""
+
+import os
+from pathlib import Path
+from typing import Union
+
+import click
+import lance
+import pandas as pd
+import pyarrow as pa
+import pyarrow.fs
+
+from bench_utils import DatasetConverter
+
+
+class BDD100kConverter(DatasetConverter):
+    def __init__(self, uri_root: Union[str, Path]):
+        super().__init__("bdd100k", uri_root)
+
+    def read_metadata(self) -> pd.DataFrame:
+        frames = []
+        for split in ["train", "val"]:
+            annotation = pd.read_json(
+                os.path.join(
+                    self.uri_root, "bdd100k", "labels", f"bdd100k_labels_images_{split}.json"
+                )
+            )
+            annotation["split"] = split
+            annotation["image_uri"] = annotation["name"].map(
+                lambda name: os.path.join(self.uri_root, "bdd100k", "images", "100k", split, name))
+            frames.append(annotation)
+
+        return pd.concat(frames)
+
+    def image_uris(self, table):
+        return table["image_uri"].to_numpy()
+
+    def get_schema(self):
+        schema = pa.schema(
```

Comment:
nit: it would be a little easier to parse the schema if labels and attributes were defined separately (so final schema fields are all on the same level and it's easy to scan.

Reply:
Not sure I understand. Could you give an example?